### PR TITLE
Rumble rename

### PR DESCRIPTION
**Pull request by ghislainfourny**
We'll rename the java packages later as they are invisible to the outside world now.

We are also moving to 1.0 (beta).

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -60,13 +60,13 @@ Once the ANTLR sources have been generated, you can compile the entire project l
 
     $ mvn clean compile assembly:single
     
-After successful completion, you can check the `target` directory, which should contain the compiled classes as well as the JAR file `jsoniq-spark-app-0.9.7-jar-with-dependencies.jar`.
+After successful completion, you can check the `target` directory, which should contain the compiled classes as well as the JAR file `jsoniq-spark-app-1.0-jar-with-dependencies.jar`.
```
Comment (CanBerker) [inline docs/install.md]: Shouldn't the jar name here be the same as the line below?
spark-rumble-1.0-jar-with-dependencies.jar

File: docs/Function library.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Function library
 
-We list here the functions supported by Sparksoniq, and introduce them by means of examples. Highly detailed specifications can be found in the [underlying W3C standard](https://www.w3.org/TR/xpath-functions-30/#func-floor), unless the function is marked as specific to JSON or Sparksoniq, in which case it can be found [here](http://www.jsoniq.org/docs/JSONiq/html-single/index.html#idm34604304).
+We list here the functions supported by Rumble, and introduce them by means of examples. Highly detailed specifications can be found in the [underlying W3C standard](https://www.w3.org/TR/xpath-functions-30/#func-floor), unless the function is marked as specific to JSON or Sparksoniq, in which case it can be found [here](http://www.jsoniq.org/docs/JSONiq/html-single/index.html#idm34604304).
```
Comment (CanBerker) [inline docs/Function library.md]: 'JSON or Rumble' -- there is a missed Sparksoniq term

File: docs/Getting started.md [hunk 3]
```diff
@@ -41,12 +41,11 @@ Create, in the same directory as Sparksoniq, a file data.json and put the follow
...
 
-In a shell, from the directory where the sparksoniq .jar lies, type, all on one line:
+In a shell, from the directory where the rumble .jar lies, type, all on one line:
 
-    spark-submit --class sparksoniq.ShellStart --master local[*] --deploy-mode client
-                 sparksoniq-0.9.7.jar
+    spark-submit --master local[*] --deploy-mode client rumble-1.0.jar --shell yes
                  
-The Sparksoniq shell appears:
+The Rumble shell appears:
```
Comment (CanBerker) [inline docs/Getting started.md]: I would recommend removing temporarily or updating the Sparksoniq banner shown below as well.

File: docs/Run on a cluster.md [hunk 4]
```diff
@@ -1,25 +1,24 @@
-# Running Sparksoniq on a cluster
+# Running Rumble on a cluster
 
-After you have tried Sparksoniq locally as explained in the getting started section, you can take Sparksoniq to a real cluster
+After you have tried Rumble locally as explained in the getting started section, you can take Sparksoniq to a real cluster
```
Comment (CanBerker) [inline docs/Run on a cluster.md]: You can take Rumble to a real cluster

File: docs/install.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
-# Installing Sparksoniq
+# Installing Rumble
 
-We show here how to install Sparksoniq from the github repository if you wish to do so (for example, to use the latest master). However, the easiest way to use Sparksoniq is to simply download the already compiled .jar files.
+We show here how to install Rumble from the github repository if you wish to do so (for example, to use the latest master). However, the easiest way to use Sparksoniq is to simply download the already compiled .jar files.
```
Comment (CanBerker) [inline docs/install.md]: the easiest way to use Rumble

File: docs/install.md [hunk 6]
```diff
@@ -30,45 +30,45 @@ Type the following commands to check that the necessary commands are available.
...
-The most straightforward to test if the above steps were successful is to run the Sparksoniq shell locally, like so:
+The most straightforward to test if the above steps were successful is to run the Rumble shell locally, like so:
 
-    $ spark-submit --class sparksoniq.ShellStart --master local[2] --deploy-mode client target/jsoniq-spark-app-0.9.7-jar-with-dependencies.jar
+    $ spark-submit --master local[2] --deploy-mode client target/spark-rumble-1.0-jar-with-dependencies.jar --shell yes
 
-The Sparksoniq shell should start:
+The Rumble shell should start:
 
     Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
```
Comment (CanBerker) [inline docs/install.md]: Removing/updating given below as well

File: docs/install.md [hunk 7]
```diff
@@ -107,12 +107,12 @@ You can try a few more queries.
     >>> 
     ( 1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
     
-This is it. Sparksoniq is step and ready to go locally. You can now move on to a JSONiq tutorial. A Sparksoniq tutorial will also follow soon.
+This is it. Rumble is step and ready to go locally. You can now move on to a JSONiq tutorial. A Sparksoniq tutorial will also follow soon.
```
Comment (CanBerker) [inline docs/install.md]: A Rumble tutorial will also follow soon

**Comment by CanBerker** [comment]
I have also noticed the following link on the website
image

**Comment by CanBerker** [comment]
Link to website needs to updated at the repo description.
image

Also the README.md of the repo needs to be updated

File: docs/Getting started.md (hunk 3 above)
Comment (ghislainfourny) [reply docs/Getting started.md]: Done

File: docs/install.md (hunk 1 above)
Comment (ghislainfourny) [reply docs/install.md]: Good catch!

File: docs/Function library.md (hunk 2 above)
Comment (ghislainfourny) [reply docs/Function library.md]: Done

File: docs/Run on a cluster.md (hunk 4 above)
Comment (ghislainfourny) [reply docs/Run on a cluster.md]: Done

File: docs/install.md (hunk 5 above)
Comment (ghislainfourny) [reply docs/install.md]: Done

File: docs/install.md (hunk 6 above)
Comment (ghislainfourny) [reply docs/install.md]: Done

File: docs/install.md (hunk 7 above)
Comment (ghislainfourny) [reply docs/install.md]: Done

**Comment by ghislainfourny** [comment]
> Link to website needs to updated at the repo description.
> image
> 
> Also the README.md of the repo needs to be updated

Done

**Comment by ghislainfourny** [comment]
> Link to website needs to updated at the repo description.
> image
> 
> Also the README.md of the repo needs to be updated

Done

**Comment by ghislainfourny** [comment]
> I have also noticed the following link on the website
> image

Done

**Pull request merged**